src/TurbulenceConvection/EDMF_functions.jl

```diff
@@ -780,12 +781,12 @@ function compute_up_tendencies!(
         detr_w = aux_up[i].detr_turb_dyn
         buoy = aux_up[i].buoy
 
-        @. tends_ρaw = -(∇f(wvec(LBC(ρaw * w_up))))
-        @. tends_ρaw +=
-            (ρaw * (I0f(entr_w) * w_en - I0f(detr_w) * w_up)) +
-            (ρ_f * Iaf(a_up) * I0f(buoy)) +
-            nh_pressure
-        tends_ρaw[kf_surf] = 0
+        @. tends_w = -(∇f(wvec(LBC(w * w_up))))
+        @. tends_w +=
+            w * I0f(entr_w) * (w_en - w_up) +
```

Comment:
We used to have `detr_w` multiplying `w_up` here. Now there is only `entr_w`. Is that on purpose?

Reply:
yes, see equation (21) in Tan et al
<img width="777" alt="Screen Shot 2022-10-09 at 6 34 39 PM" src="https://user-images.githubusercontent.com/24443883/194784628-5de5a1ea-2bc4-4591-a795-3e7b187352cf.png">

Reply:
So the detrainment was only there through density changes?